tests/unit/config/test_configfiles.py

```diff
@@ -134,32 +134,103 @@ def test_yaml_oserror(fake_save_manager, config_tmpdir):
     assert error.traceback is None
 
 
-class TestConfigPy:
+class ConfPy:
 
-    """Tests for ConfigAPI and read_config_py()."""
+    """Helper class to get a confpy fixture."""
+
+    def __init__(self, tmpdir, filename: str = "config.py"):
+        self._file = tmpdir / filename
+        self.filename = str(self._file)
+
+    def write(self, *lines):
+        text = '\n'.join(lines)
+        self._file.write_text(text, 'utf-8', ensure=True)
+
+    def read(self):
+        """Read the config.py via configfiles and check for errors."""
+        api = configfiles.read_config_py(self.filename)
+        assert not api.errors
+
+    def write_qbmodule(self):
+        self.write('import qbmodule',
+                   'qbmodule.run(config)')
+
+
+class TestConfigPyModules:
 
     pytestmark = pytest.mark.usefixtures('config_stub', 'key_config_stub')
 
-    class ConfPy:
+    @pytest.fixture
+    def confpy(self, tmpdir):
+        return ConfPy(tmpdir)
+
+    @pytest.fixture
+    def qbmodulepy(self, tmpdir):
+        return ConfPy(tmpdir, filename="qbmodule.py")
+
+    @pytest.fixture(autouse=True)
+    def restore_sys_path(self):
+        old_path = sys.path.copy()
```

Comment:
Oh, missing `sys` import.

Reply:
hmm, it's weird this is passing on my machine but not on travis (there is a sys import on the top of the file). I'll try adding a import sys in the fixure (are fixures isolated like that)?

Reply:
Ah, I think I know what's happening: I removed the import in ef1c83862b8166066a860fbfac93a19ec26bb8c3 and Travis tests it against the latest master, while your local branch is on an older master. So the right solution is to rebase your local branch on the latest master, remove this, and re-add it at the top :wink: